src/main/java/edu/cornell/kfs/krad/antivirus/ScanResult.java

```diff
@@ -1,4 +1,4 @@
-package edu.cornell.kfs.krad.service;
+package edu.cornell.kfs.krad.antivirus;
```

Comment:
should this be edu.cornell.kfs.krad.antivirus.service?

Reply:
I wasn't sure whether this enum should be in the "service" package or not, but I can move it there to reduce confusion.